docs/source/tutorial/introduction.md

```diff
@@ -55,10 +55,10 @@ Now the fun begins! First, you'll need to install our developer tools:
 - [Apollo DevTools for Chrome (suggested)](https://chrome.google.com/webstore/detail/apollo-client-developer-t/jdkknkkbebbapilgoeccciglkfbmbnfm) : Our Chrome extension giving you full visibility into your client.
 - [Apollo VSCode (suggested)](https://marketplace.visualstudio.com/items?itemName=apollographql.vscode-apollo): Our editor integration that offers intelligent autocomplete, metrics, and more.
 
-Next, in your terminal, clone [this repository](https://github.com/apollographql/fullstack-tutorial):
+Next, in your terminal, clone [this repository](https://github.com/apollographql/fullstack-tutorial.git):
 
 ```bash
-git clone https://github.com/apollographql/fullstack-tutorial/
+git clone https://github.com/apollographql/fullstack-tutorial.git
```

Comment:
This should work either way.  If you're having problems with this, please share a screenshot of your attempt and some reproduction information.